### ISSUE TITLE
Autowiring definitions

### ISSUE THREAD

**Issue by mnapoli** [mnapoli/fluent-symfony#1]
Add a new `autowire()` helper:

```php
return [
    'mailer' => autowire(Mailer::class),
];
```

**Comment by rejinka** [comment]
The result should then be like the following?

```
# app/config/services.yml
services:
    mailer:
        class: Mailer
        autowire: true
```

If yes, I'd like to make a PR.

**Comment by mnapoli** [comment]
Yes that's exactly that! The PR would be very welcome!

**Comment by mnapoli** [comment]
Implemented by @rejinka in #4 \o/
